code/go/0chain.net/smartcontract/minersc/dkg.go

```diff
@@ -703,6 +696,13 @@ func (msc *MinerSmartContract) shareSignsOrShares(t *transaction.Transaction,
 			"invalid state: decoding miners MPK: %v", err)
 	}
 
+	// We may use != instead of < here? @kenwes13
+	if len(sos.ShareOrSigns) < len(mpks.Mpks)-1 {
```

Comment:
Use `if len(sos.ShareOrSigns) < dmn.K-1`.

Reply:
Never mind, I'll update it in fix/vc-true-stuck branch. Let's see how would this go with fix/vc-true-struck branch.